README.md

```diff
@@ -73,6 +73,39 @@ Guide](https://solr.apache.org/guide/solr/latest/getting-started/solr-tutorial.h
 - Slack: Solr Community Channel.  Sign up at https://s.apache.org/solr-slack
 - IRC: `#solr` on [libera.chat](https://web.libera.chat/?channels=#solr)
 
+## Developer Documentation
+
+Solr uses Gradle for its build system. Here some useful hints to build and run Solr locally:
+
+- To build a Solr dev distribution for your local OS run:
```

Comment:
"local OS run"  --> Not sure I get the OS term?

Reply:
I have removed entirely the `for your local OS run` sentence.